eng/build.yml

```diff
@@ -87,8 +87,7 @@ jobs:
         condition: succeededOrFailed()
         env:
             SYSTEM_ACCESSTOKEN: $(System.AccessToken)
-            HelixAccessToken: ''
-            RunAoTTests: 'false'
```

Comment:
I don't think the RunAoTTests environment variable should have been removed here. Looks like a bad merge maybe? @TanayParikh to confirm.

Reply:
Thanks @marcpopMSFT, that should definitely remain. 

Looks like @v-wuzhai has now reverted this bit via https://github.com/dotnet/sdk/pull/25403/commits/fa9cf3350054f9d52ab11bbf46566ff1bbf8f3db.